### DISCUSSION TITLE
RustDesk prevents Turn off screen (Win11)

### DISCUSSION THREAD

**Discussion by mitrios-hub** [rustdesk/rustdesk#8572] [Q&A] [answered]
**powercfg /requests** shows that the RD process requires to keep screen ON. After restart RD service this issue was fixed. Is this bug? I need to keep rustdesk service running but screen must be turned off by Windows on timer.

RD ver: 1.2.2 2023-08-22 01:58

**Comment by rustdesk** [comment, marked as answer]
we do not have such request in 1.2.2.

but in 1.2.6, we do have, https://github.com/rustdesk/rustdesk/discussions/8473#discussioncomment-9891505, and we fixed manual sleep/hibernate and ready in nightly build, you still see in the powercfg within connection (the other tool, e.g. anydesk also has), but it disappear after disconnection.

![mmexport1719921521714](https://github.com/rustdesk/rustdesk/assets/71636191/0fd10622-3074-42b7-94f9-27cb224b029d)
